### ISSUE TITLE
Confused about the sampling algorithm

### ISSUE THREAD

**Issue by Morefre** [NVlabs/I2SB#4]
I understand the theories and how they come up with the algorithm. However, the resulting algorithm is very confusing. Just by looking at the algorithm 1 and 2, what IS2B actually does is training a diffusion model using VE-SDE as in equation (12). The only difference from standard VE-SDE is that they use a different forward process to generate data for training the network as in equation (11). That's it, nothing more. 

It's surprising this method will work in practice since the data generation process is different from the defined forward SDE. Will the method still work if we design another data generation process instead of equation (11). Will  the method still work if we change equation (12) to the training formula for VP-SDE and change the DDPM posterior formula in algorithm 2  to that of the VP-SDE? 

So, basically I don't know how to understand I2SB correctly. Any discussion or correction will be appreciated !

<img width="770" alt="image" src="https://github.com/NVlabs/I2SB/assets/101572174/4c01b51d-36e5-4988-a7c3-c67f624e7b35">
<img width="356" alt="image" src="https://github.com/NVlabs/I2SB/assets/101572174/a8d7daaf-e74b-4fff-aef0-578cdc7adc59">
<img width="361" alt="image" src="https://github.com/NVlabs/I2SB/assets/101572174/fdf00de6-2ea9-4b2d-8b00-46d479ee531a">
